api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -388,43 +389,43 @@ public void sendFileLengthsEgressRemediationEmail(DbUser dbUser, EgressRemediati
   }
 
   @Override
-  public void sendPublishWorkspaceEmail(
-      DbWorkspace workspace, List<DbUser> owners, FeaturedWorkspaceCategory publishCategory)
+  public void sendPublishCommunityWorkspaceEmails(DbWorkspace workspace, List<DbUser> owners)
       throws MessagingException {
-
-    String supportEmail = workbenchConfigProvider.get().mandrill.fromEmail;
-
-    sendWithRetries(
-        ownersEmailList(owners),
-        Collections.singletonList(supportEmail),
-        "Your AoU Researcher Workbench workspace has been published",
-        String.format(
-            "Publish workspace email for workspace '%s' (%s) sent to owners %s",
-            workspace.getName(), workspace.getWorkspaceNamespace(), ownersForLogging(owners)),
-        buildHtml(
-            PUBLISH_WORKSPACE_RESOURCE,
-            publishUnpublishWorkspaceSubstitutionMap(
-                workspace,
-                featuredWorkspaceCategoryAsDisplayString(publishCategory),
-                supportEmail)));
+    sendPublishUnpublishWorkspaceEmails(
+        workspace, owners, PUBLISH_COMMUNITY_WORKSPACE_RESOURCE, "Publish", "published");
   }
 
   @Override
-  public void sendUnpublishWorkspaceByAdminEmail(DbWorkspace workspace, List<DbUser> owners)
+  public void sendAdminUnpublishWorkspaceEmails(DbWorkspace workspace, List<DbUser> owners)
       throws MessagingException {
+    sendPublishUnpublishWorkspaceEmails(
+        workspace, owners, UNPUBLISH_WORKSPACE_RESOURCE, "Unpublish", "unpublished");
+  }
 
-    String supportEmail = workbenchConfigProvider.get().mandrill.fromEmail;
-
-    sendWithRetries(
-        ownersEmailList(owners),
-        Collections.singletonList(supportEmail),
-        "Your AoU Researcher Workbench workspace has been Unpublished",
-        String.format(
-            "Unpublish workspace by admin email for workspace '%s' (%s) sent to owners %s",
-            workspace.getName(), workspace.getWorkspaceNamespace(), ownersForLogging(owners)),
-        buildHtml(
-            UNPUBLISH_WORKSPACE_RESOURCE,
-            publishUnpublishWorkspaceSubstitutionMap(workspace, "", supportEmail)));
+  private void sendPublishUnpublishWorkspaceEmails(
+      DbWorkspace workspace,
+      List<DbUser> owners,
+      String emailResource,
+      String presentTenseCapitalized,
```

Comment:
How about actionPresentTense,Capitalized String actionPastTense 
Just so we know whats in pastTense and presentTense